lib/dune

```diff
@@ -6,10 +6,6 @@
 (rule
  (targets asset.ml asset.mli)
  (deps
-  ; Not specifying dependencies on main.css on purpose so we can skip the generation
-  ; of the CSS file if we only want to run the server.
-  ; Uncomment in development
-  ../asset/main.css
```

Comment:
This was removed to allow the build to succeed. See https://github.com/mirage/mirage-www/issues/796.